test/test_writer.jl

```diff
@@ -121,6 +123,62 @@ include("external_unzippers.jl")
     end
 end
 
+@testset "Writer compat with ZipStreams" begin
+    # setup test env for ZipStreams
```

Comment:
`ZipStreams` needs an old version of `TranscodingStreams`, so I'm using `Malt` to run it in an isolated environment.